server/.env

```diff
@@ -1 +1,3 @@
-TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Zeeshan, Saad"
+TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Zeeshan, Saad, Liang"
```

Comment:
should use .gitignore not to have it in repo. i think that's the purpose of env so that it won't go public IMO. @aeciorc  or in this case it is fine to have it on public ?

Reply:
@sk-izsk That's a good point, we addressed this on Saad's PR for db setup